static_src/routes.js

```diff
@@ -93,7 +93,8 @@ export function space(orgGuid, spaceGuid, next) {
   spaceActions.fetch(spaceGuid);
   serviceActions.fetchAllInstances(spaceGuid);
   userActions.changeCurrentlyViewedType('space_users');
-  userActions.fetchSpaceUsers(spaceGuid);
+  userActions.fetchOrgUsers(orgGuid);
```

Comment:
to be consistent, is there a reason why this wasn't also renamed to be `fetchOrgUserRoles`?